libs/ui/elements/src/lib/metadata-info/metadata-info.component.ts

```diff
@@ -37,6 +42,24 @@ export class MetadataInfoComponent {
     return array
   }
 
+  get updateFrequency(): string {
+    if (
+      (this.metadata.updateFrequency as UpdateFrequencyCustom).updatedTimes !==
```

Comment:
I think the assertion here could be:
```
if (this.metadata.updateFrequency instanceof Object)
```

Then TS should infer that we're indeed in the case of an `UpdateFrequencyCustom` ad you shouldn't have to cast it again :)

Reply:
You're, right, much more readable... and typescript is pretty smart, actually.

Reply:
annoying but smart ;)